### ISSUE TITLE
Cannot move through long lists item by item with VoiceOver

### ISSUE THREAD

**Issue by charypar** [redbadger/pride-london-app#410]
## Description of issue

On iOS with VoiceOver, stepping through items truncates the list. It is necessary to use the scroll gesture to force the list to pick up further items and make them available for stepping through.

Works fine on Android.

## Expected behaviour

User can step through the entire list using the back and forward gestures, or the skip gesture (e.g. in headline mode)

## Steps to reproduce

1. Open the events list with VoiceOver on
2. step through the items until the 'next' gesture does not get you any further
3. swipe up with three fingers (scrolling to the "next page"), new items are revealed and can be stepped through

## OS / Browser / Device model

iOS 11.1, iPhone 7

**Comment by RGBboy** [comment]
@charypar is this something we can fix? Or do we need to wait for React Native to fix this?

**Comment by charypar** [comment]
There might be some workaround, but I'm not sure. AirBnB have a fix for it in their RN fork and it looks like the original issue was closed...
